streampark-console/streampark-console-webapp/src/views/flink/home/components/Modal.vue

```diff
@@ -99,14 +99,14 @@
       flinkName: formValue.flinkName,
       flinkHome: formValue.flinkHome,
     });
-    const checkResp = parseInt(resp.data);
-    if (checkResp != 0) {
+    const checkResp = resp.data;
+    if (checkResp != 'ok') {
```

Comment:
Here is how the front-end (FE) use enums to adapt to the values returned by the back-end (BE), You can refer to it:

https://github.com/apache/incubator-streampark/blob/dev/streampark-console/streampark-console-webapp/src/enums/flinkEnum.ts